platforms/documentation/docs/src/docs/userguide/reference/other-topics/init_scripts.adoc

```diff
@@ -138,12 +119,22 @@ include::{snippetsPath}/initScripts/externalDependency/tests-common/externalInit
 include::{snippetsPath}/initScripts/externalDependency/tests-common/externalInitDependency.out[]
 ----
 
-[[sec:init_script_plugins]]
-== Applying plugins
+[[sec:init_plugins]]
+== Init plugins
+
+An init plugin is a Gradle plugin whose target type is `Plugin<Gradle>` and is applied in an init script.
+
+Unlike project (`Plugin<Project>`) or settings (`Plugin<Settings>`) plugins, an init plugin attaches to the Gradle object during the initialization phase, before the settings and projects are configured:
 
-Plugins can be applied to init scripts like a Gradle build script or a Gradle settings file.
+- *Scope*: Runs at init; can access `settingsEvaluated`, `projectsLoaded`, `beforeProject`/`afterProject` callbacks to safely interact with settings/projects when they exist.
+- *Distribution*: Applied from an init script (`~/.gradle/init.d/*.init.gradle(.kts)` or `$GRADLE_HOME/init.d`) so it affects every build on that machine/agent.
```

Comment:
Is it worth getting more specific here? Initscripts may not affect builds such as those done by TestKit, as they use a separate Gradle User Home. Additionally, if different users run different builds, they won't necessarily both get the initscripts.